src/Client/ClientBase.cpp

```diff
@@ -2384,23 +2384,23 @@ void ClientBase::runNonInteractive()
         return;
     }
 
-    String text;
-    if (config().has("query"))
+    if (!queries.empty())
     {
-        text += config().getRawString("query"); /// Poco configuration should not process substitutions in form of ${...} inside query.
+        for (const auto & query : queries)
+        {
+            bool result = query_fuzzer_runs ? processWithFuzzing(query) : processQueryText(query);
+            if (!result)
+                return;
+        }
     }
     else
     {
         /// If 'query' parameter is not set, read a query from stdin.
         /// The query is read entirely into memory (streaming is disabled).
         ReadBufferFromFileDescriptor in(STDIN_FILENO);
+        String text;
```

Comment:
The `else` case (l. 2396) no longer calls `processWithFuzzing/processQuery`. Intended?